### PR TITLE
Updating color pallete

### PR DESCRIPTION
**Pull request by Krymancer**
So i see your project uses some colors that are not in the palettes in the [catppuccin](https://github.com/catppuccin/catppuccin) readme.

So i updated the palette and started a concpet of a way to maybe suport mutiple themes.

I guess the way i doi it maybe not work because insomnia plugin hub only acpect a theme por package (i think, so im open to ideias as well)

I tried be consise and faithful ao thema criado, but some colors in the palette dont have the hex codes you used, so i change them.

Hope we can update this theme publish into the catppucin and the insmnia plugin hub.

### CODE REVIEW

File: themes/catppuccin.js [hunk 1]
```diff
@@ -1,39 +1,50 @@
+import colors from "./colors";
```
Comment (opakholis) [inline themes/catppuccin.js:1]: heyo @Krymancer, love your work, but it seems this is still causing the error, and if you don't mind I'll fix it later

Comment (Krymancer) [reply themes/catppuccin.js]: Sorry about that, i make a quick fix and i think it's working now

Comment (Krymancer) [reply themes/catppuccin.js]: So i just make an update to have all the themes exported, but i'm with a bug for text in the frappe (light version).
I think maybe chose other colors to all themes or make a new variant based i the theme is light or dark.

image
Rn i'm using surface color thats works well in dark themes, but in frappe is broken
image

Comment (opakholis) [reply themes/catppuccin.js]: LGTM🔥

**Comment by opakholis** [comment]
Merged. thanks!

**Pull request merged**